### DISCUSSION TITLE
Organization profile showing introductory help to admins?

### DISCUSSION THREAD

**Discussion by eritbh** [community/community#38112] [Other Feature Feedback, Questions, & Ideas]
Hi, I'm the owner of an existing organization and for some reason viewing the organization's "profile" page looks like this:

![image](https://user-images.githubusercontent.com/4165301/199621534-6392a438-bdf7-4d14-b35f-6f0378f9bc4c.png)

The introductory help stuff here isn't helpful at all for an organization that's been around a while and already has several existing repos. There also doesn't seem to be any way to dismiss it without following each individual step in a demo repo which I don't want to create in this org. How can I make this go away and just see the normal profile view with a list of repositories and stuff like normal?

**Comment by ossanna16** [comment]
Hi there @eritbh and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by eritbh** [comment]
This is still an issue, and it's been making it pretty annoying to access repositories under the organization. Anyone?

**Reply by alexlato22** [reply]
@eritbh Thanks for the feedback! You can always click "hide the tasks we've suggested" on the right hand side to hide them. 
![199621534-6392a438-bdf7-4d14-b35f-6f0378f9bc4c](https://user-images.githubusercontent.com/102535292/213691717-55cfe9b6-0647-4de9-85fb-757c8922696d.png)

**Reply by eritbh** [reply]
Thank you @alexlato22! Not sure how I missed that link this whole time... >.>

**Comment by jfullstackdev** [comment]
I'm not seeing it on my org

![image](https://user-images.githubusercontent.com/47092464/202896308-f76a3f9f-d178-4919-98d5-cd7b58f0bf87.png)
